modules/python/src2/pycompat.hpp

```diff
@@ -54,9 +54,19 @@
 #define PyInt_FromLong PyLong_FromLong
 #define PyNumber_Int PyNumber_Long
 
-
+#if defined(_WIN32)
+static inline PyObject* PyString_FromString(const char* u)
+{
+    return PyUnicode_DecodeMBCS(u, strlen(u), nullptr);
```

Comment:
3.4 branch should compile with C++98, so (unfortunately) use `NULL` instead of `nullptr`.